source/PreferencesPanel.cpp

```diff
@@ -583,27 +586,49 @@ void PreferencesPanel::DrawPlugins()
 	const Color &back = *GameData::Colors().Get("faint");
 	const Color &medium = *GameData::Colors().Get("medium");
 	const Color &bright = *GameData::Colors().Get("bright");
+	const Color &error = *GameData::Colors().Get("bright error");
+
+	const Sprite *box[2] = { SpriteSet::Get("ui/unchecked"), SpriteSet::Get("ui/checked") };
 
 	const int MAX_TEXT_WIDTH = 230;
 	Table table;
 	table.AddColumn(-115, {MAX_TEXT_WIDTH, Truncate::MIDDLE});
-	table.SetUnderline(-120, 120);
+	table.SetUnderline(-120, 100);
 
 	int firstY = -238;
-	table.DrawAt(Point(-130, firstY));
+	// Table is at -110 while checkbox is at -130
+	table.DrawAt(Point(-110, firstY));
```

Comment:
Currently `Table` only knows how to draw text. Would it make sense to support drawing a sprite centered in the active column? This would align `Table` a bit more with HTML tables (minus the bits about automatic bounds).

Reply:
I’m not sure.  I am open to changes.  I usually only write CLI programs for this UI code was me throwing everything at a wall to see what sticks.